flink-tests/src/test/java/org/apache/flink/test/checkpointing/RescaleCheckpointManuallyITCase.java

```diff
@@ -150,54 +137,33 @@ public void testCheckpointRescalingKeyedState(boolean scaleOut) throws Exception
                 numberKeys,
                 numberElements2,
                 numberElements + numberElements2,
-                client,
+                miniCluster,
                 checkpointPath);
     }
 
-    private static String runJobAndGetCheckpoint(
+    private String runJobAndGetCheckpoint(
             int numberKeys,
             int numberElements,
             int parallelism,
             int maxParallelism,
-            ClusterClient<?> client,
-            File checkpointDir)
+            MiniCluster miniCluster)
```

Comment:
Why do we have to change `ClusterClient` to `MiniCluster` here? Does it needed by `getLatestCompletedCheckpointPath`?

Reply:
Yes, `getLatestCompletedCheckpointPath` needs `MiniCluster ` parameter.
```Java
public static Optional<String> getLatestCompletedCheckpointPath(
            JobID jobID, MiniCluster cluster){}
```